lib/Onyx.js

```diff
@@ -775,6 +805,7 @@ function applyDecorators() {
     merge = decorate.decorateWithMetrics(merge, 'Onyx:merge');
     mergeCollection = decorate.decorateWithMetrics(mergeCollection, 'Onyx:mergeCollection');
     getAllKeys = decorate.decorateWithMetrics(getAllKeys, 'Onyx:getAllKeys');
+    initializeWithDefaultKeyStates = decorate.decorateWithMetrics(initializeWithDefaultKeyStates, 'Onyx:defaults');
```

Comment:
Track the time added by `initializeWithDefaultKeyStates` 
We don't need to add `addAllSafeEvictionKeysToRecentlyAccessedList` as it's already covered by `getAllKeys`